core/src/main/java/io/undertow/websockets/core/WebSocketChannel.java

```diff
@@ -82,6 +83,8 @@
      */
     private final Set<WebSocketChannel> peerConnections;
 
+    //TODO: possibly make it i18n msg ?
```

Comment:
Yes, please add the message to WebSocketMessages, following the same that is done for WebSocketMessages.messageToBig

Reply:
Ok